Utilities/WSIWindow/.clang-format

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Rene, you shouldn't need this custom version of .clang-format -- the tool should pick up the one in the root directory.  If for some reason you DO need one here, just copy that one in and re-run clang-format.